### PR TITLE
Updated headings, button text, and playwright tests.

### LINKED ISSUE #838: [Update] Update Heading & Button Text
- [x] Change “Resolve errors (1 of 2)” to “Resolve errors (syntax)”
- [x] Change “Resolve errors (2 of 2) to “Resolve errors (logic)”
- [x] Change button between steps to “Continue to next step”
- [x] Change button text between resolve errors pages to “Continue”
- [x] Error notifications that refer to “save and continue” (designed to address instances where users try to click the continue button without completing the requirements of the step), must be updated to “continue to the next step”
- [x] Update Playwright tests

### PR DESCRIPTION
**Pull request by ojbravo**
Closes #838. All Playwright tests pass.

### CODE REVIEW

File: src/pages/PointOfContact/index.tsx [hunk 1]
```diff
@@ -244,7 +244,7 @@ function PointOfContact(): JSX.Element {
           alertHeading={
             hasFormRegexErrors
               ? 'There was a problem updating your point of contact information'
-              : 'You must provide all required point of contact information to save and continue'
+              : 'You must provide all required point of contact information to continue to next step'
```
Comment (natalia-fitzgerald) [inline src/pages/PointOfContact/index.tsx:247]: Update:
'There was a problem updating your point of contact information'

File: src/pages/TypesFinancialInstitutions/index.tsx [hunk 2]
```diff
@@ -135,7 +135,7 @@ function TypesFinancialInstitutions(): JSX.Element {
           alertHeading={
             hasOtherFormError
               ? 'There was a problem updating your type of financial institution'
-              : 'You must select your type of financial institution to save and continue'
+              : 'You must select your type of financial institution to continue to next step'
```
Comment (natalia-fitzgerald) [inline src/pages/TypesFinancialInstitutions/index.tsx]: Update: 
'There was a problem updating your type of financial institution'

File: src/pages/PointOfContact/index.tsx [hunk 3]
```diff
@@ -256,7 +256,7 @@ function PointOfContact(): JSX.Element {
             You are required to complete all fields with the exception of the
             street address lines labeled optional. Your point of contact
             information will not be saved until you provide all required
-            information and click &quot;Save and continue.&quot;
+            information and click &quot;Continue to next step.&quot;
```
Comment (natalia-fitzgerald) [inline src/pages/PointOfContact/index.tsx:259]: Update: 
'You are required to complete all fields with the exception of the street address lines labeled optional. Your point of contact information will not be saved until you provide all required information and continue to the next step.'

File: src/pages/PointOfContact/index.tsx (hunk 1 above)
Comment (billhimmelsbach) [reply src/pages/PointOfContact/index.tsx:247]: Heyo @natalia-fitzgerald, these are handled in my PR over here:
https://github.com/cfpb/sbl-frontend/pull/855/files

File: src/pages/PointOfContact/index.tsx (hunk 3 above)
Comment (billhimmelsbach) [reply src/pages/PointOfContact/index.tsx:259]: Heyo @natalia-fitzgerald, these are handled in my PR over here:
https://github.com/cfpb/sbl-frontend/pull/855/files

File: src/pages/TypesFinancialInstitutions/index.tsx (hunk 2 above)
Comment (billhimmelsbach) [reply src/pages/TypesFinancialInstitutions/index.tsx]: Heyo @natalia-fitzgerald, this change was already made when your review went up. GitHub will show you a "refresh" option while you're doing your review.

**Review by billhimmelsbach — DISMISSED** [review]
Looks good and it was approved by Natalia in SBL Frontend chat. 👍

**Review by billhimmelsbach — DISMISSED** [review]
Natalia approved these change in our channel, and the code looks good. Thanks!

**Comment by natalia-fitzgerald** [comment]
I reviewed the changes on dev and everything looks good. The changes are approved.

**Review by natalia-fitzgerald — APPROVED** [review]
@ojbravo 
The changes look good to go. Later today I will send updated content for line 255 but I don't want this change to block merging this PR now. Thanks!

**Pull request merged**